From ca04863992fe1187788832396f7484c0fb3e3bd1 Mon Sep 17 00:00:00 2001
From: Salvatore Bonaccorso <carnil@debian.org>
Date: Tue, 3 Jul 2012 21:51:32 +0200
Subject: [PATCH] Work with changed output from subversion 1.7

Patch t/corelist.pm to work with changed output of svnnotify. Fix FTBFS
"tests failed".

Closes: #678791
---
 t/coretests.pm |    1 +
 1 file changed, 1 insertion(+)

diff --git a/t/coretests.pm b/t/coretests.pm
index 0be09e2..f167eeb 100644
--- a/t/coretests.pm
+++ b/t/coretests.pm
@@ -178,6 +178,7 @@ sub _test {
     open $TESTER, '-|', _build_command($command, %args);
     while (<$TESTER>) {
 	chomp;
+	next if ( /^Updating '.+':/ );
 	if ( /^At revision (\d+)\./ ) {
 	    ok ( $1 == $args{revision} , "No changes in $prefix at revision: "
 	    	. $args{revision} );
-- 
1.7.10

